### PR TITLE
Correct SGB packet delay durations

### PR DESCRIPTION
**Pull request by pinobatch**
Based on research in gbdev Discord:
- zlago provided a test ROM showing that 2 μs pulses work
- GenericHeroGuy showed how to get SGB not to check for a packet for 4 frames
- Fix two comma splices

(New text uses a newline after each clause.)

### CODE REVIEW

**Comment by nitro2k01** [comment]
In my opinion it's too early to remove the recommendation for longer wait times. While the test ROM shows that a minimal delay time can work, and probably works most of the time, the recommendation may exist to account for some corner case that we haven't seen yet in testing.

Also, isn't it more relevant to point out that the timing recommendation comes from Nintendo's programming manual, than to say what licensed games are doing de facto?

**Comment by pinobatch** [comment]
I phrased it in terms of what licensed games do because I thought we were trying to limit how much we cite the manual. I'll see how I can reword it to state that 5 on 15 off is still the current best practice.

**Review by avivace — APPROVED** [review]

File: src/SGB_Command_Packet.md [hunk 1]
```diff
@@ -8,30 +8,36 @@ Game Boy keyboard matrix (which still works).
...
+Time   0        50        100       150
 ```
 
-Data and reset pulses must be kept LOW for at least 5us. P14 and P15
-must be kept both HIGH for at least 15us between any pulses. Obviously,
-it'd be no good idea to access the JOYPAD register during the transfer,
+The boot ROM and licensed software keep data and reset pulses LOW for at least 5 μs and leave P14 and P15 HIGH for at least 15 μs after each pulse.
+Though the hardware is capable of receiving pulses and spaces as short as 2 μs (as tested using [sgb-speedtest]),
+following the common practice of 5-cycle pulses and 15-cycle spaces may improve reliability in some corner case that the community has not yet discovered.
+Obviously, it'd be no good idea to access the JOYPAD register during the transfer,
```
Comment (ISSOtm) [inline src/SGB_Command_Packet.md:25]:
```suggestion
Obviously, it'd be no good idea to access [the joypad register](<#FF00 — P1/JOYP: Joypad>) during the transfer,
```

File: src/SGB_Command_Packet.md [hunk 2]
```diff
@@ -8,30 +8,36 @@ Game Boy keyboard matrix (which still works).
...
 
+[sgb-speedtest]: https://github.com/zlago/sgb-speedtest/
+
 ## Transferring Packets
 
 Each packet is invoked by a RESET pulse, then 128 bits of data are
 transferred (16 bytes, LSB of first byte first), and finally, a
-"0" bit must be transferred as stop bit. The structure of normal
-packets thus is:
+"0" bit must be transferred as stop bit.
```
Comment (ISSOtm) [inline src/SGB_Command_Packet.md:37]:
```suggestion
"0" bit must be transferred as a stop bit.
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Great!

**Review by avivace — APPROVED** [review]

File: src/SGB_Command_Packet.md [hunk 3]
```diff
@@ -8,30 +8,36 @@ Game Boy keyboard matrix (which still works).
...
-to LOW, this will reset and start the SNES packet receiving program.
+to LOW; this will reset and start the ICD2 packet receiving circuit.
 Data is then transferred (LSB first), setting P14=LOW will indicate a
 "0" bit, and setting P15=LOW will indicate a "1" bit. For example:
 
 ```
     RESET  0   0   1   1   0   1   0
 P14  --_---_---_-----------_-------_--...
 P15  --_-----------_---_-------_------...
+Time   0        50        100       150
```
Comment (ISSOtm) [inline src/SGB_Command_Packet.md:19]:
```suggestion
       ↑        ↑         ↑         ↑
Time   0        50        100       150
```

File: src/SGB_Command_Packet.md [hunk 4]
```diff
@@ -8,30 +8,36 @@ Game Boy keyboard matrix (which still works).
...
 ```
 
-Data and reset pulses must be kept LOW for at least 5us. P14 and P15
-must be kept both HIGH for at least 15us between any pulses. Obviously,
-it'd be no good idea to access the JOYPAD register during the transfer,
+The boot ROM and licensed software keep data and reset pulses LOW for at least 5 μs and leave P14 and P15 HIGH for at least 15 μs after each pulse.
```
Comment (ISSOtm) [inline src/SGB_Command_Packet.md:22]:
```suggestion
[The boot ROM](<#Super Game Boy (SGB, SGB2)>) and licensed software keep data and reset pulses LOW for at least 5 μs and leave P14 and P15 HIGH for at least 15 μs after each pulse.
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: src/SGB_Command_Packet.md (hunk 3 above)
Comment (pinobatch) [reply src/SGB_Command_Packet.md:19]: The arrows are supposed to be 50 M-cycles apart, with each character representing 5 M-cycles.

Comment (ISSOtm) [reply src/SGB_Command_Packet.md:19]: Fixed, I think?

File: src/SGB_Command_Packet.md [hunk 5]
```diff
@@ -8,30 +8,37 @@ Game Boy keyboard matrix (which still works).
...
+to LOW; this will reset and start the ICD2 packet receiving circuit.
 Data is then transferred (LSB first), setting P14=LOW will indicate a
 "0" bit, and setting P15=LOW will indicate a "1" bit. For example:
 
 ```
     RESET  0   0   1   1   0   1   0
 P14  --_---_---_-----------_-------_--...
 P15  --_-----------_---_-------_------...
+       ↑         ↑         ↑         ↑
+Time   0       50         100       150
```
Comment (ISSOtm) [inline src/SGB_Command_Packet.md:20]:
```suggestion
Time   0         50       100       150
```
Somehow there's been a mixup. My bad.

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Looks good to me!

**Review by ISSOtm — APPROVED** [review]
